synapse/lib/node.py

```diff
@@ -485,10 +485,6 @@ async def _getTagDelEdits(self, tag, init=False):
             raise s_exc.IsRuntForm(mesg='Cannot delete tags from runt nodes.',
                                    form=self.form.full, tag=tag)
 
-        curv = self.tags.get(name, s_common.novalu)
-        if curv is s_common.novalu:
```

Comment:
I think you could still use this check to determine whether `name` is added to the list of tags to delete rather than always appending that edit.